Source/svg/SVGParser.swift

```diff
@@ -704,7 +704,13 @@ open class SVGParser {
             cleanedHexString = "\(x[0])\(x[0])\(x[1])\(x[1])\(x[2])\(x[2])"
         }
         var rgbValue: UInt32 = 0
-        Scanner(string: cleanedHexString).scanHexInt32(&rgbValue)
+        if #available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *) {
+            if let scannedInt = Scanner(string: cleanedHexString).scanUInt64(representation: .hexadecimal) {
```

Comment:
You can match two conditions into one 

`if #available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *), let scannedInt = Scanner(string: cleanedHexString).scanUInt64(representation: .hexadecimal) `

Reply:
Done